test/Select.spec.tsx

```diff
@@ -157,13 +157,13 @@ describe('Select', () => {
         it(`should be true in the element if ${property} prop is true`, async () => {
           render(<Select items={[{ label: 'foo', value: 'foo' }]} {...{ [property]: true }} />);
           const select = await findByQuerySelector('vaadin-select');
-          expect(select[property]).to.be.ok;
+          expect(select[property]).to.be.true;
```

Comment:
This test currently doesn't pass due to https://github.com/vaadin/web-components/issues/8398